montecarlo/README.md

```diff
@@ -148,12 +149,12 @@ where
     PAX_VERSION: pax (also fax if not specified above) GitHub release number (https://github.com/XENON1T/pax/releases)
     SR #: Science run numer (0, 1), adjusts some physics parameters
     SAVE_WAVEFORMS: Flag to save raw waveforms (disk space intensive); 0 - off (default), 1 - on
-    PREINIT_MACRO: (Optional) name of macro to use for Geant4 preinit (defaults to preinit_TPC.mac)
-    PREINIT_BELT: (Optional) name of macro for setting up calibration belts (defaults to preinit_B_none.mac or depending on MC_CONFIG)
-    PREINIT_EFIELD: (Optional) name of macro for varying e-field in NEST (defaults to preinit_EF_C15kVA4kV.mac)
-    OPTICAL_SETUP: (Optional) name of macro to use for Geant4 optical setup (defaults to setup_optical_S1.mac)
-    SOURCE_MACRO: (Optional) name of macro to run in Geant4 (defaults to run_<MC_CONFIG>.mac)
-    EXPERIMENT: (Optional) name of experiment (XENON1T, XENONnT) to implement (defaults to XENON1T) 
+    PREINIT_MACRO: name of macro to use for Geant4 preinit (defaults to preinit_TPC.mac)
+    PREINIT_BELT: name of macro for setting up calibration belts (defaults to preinit_B_none.mac or depending on MC_CONFIG)
+    PREINIT_EFIELD: name of macro for varying e-field in NEST (defaults to preinit_EF_C15kVA4kV.mac)
+    OPTICAL_SETUP: name of macro to use for Geant4 optical setup (defaults to setup_optical_S1.mac)
+    SOURCE_MACRO: name of macro to run in Geant4 (defaults to run_<MC_CONFIG>.mac)
+    EXPERIMENT: name of experiment (XENON1T, XENONnT) to implement (defaults to XENON1T) 
```

Comment:
Why deleting the `optional`here?

Reply:
because it doesn’t work if you don’t give the option.

Reply:
This isn't correct. I just tried and the workflow starts without these options.

Reply:
not when you run locally on midway.
try:  ./run_sim.sh 1 G4p10 Cryostat_K40 100 v3.0.0 head head 0 0 XENONnT . 
if you leave out the optional inputs the experiments isn't set correctly.

Reply:
You're right (and we can merge your fix as it is). But I would then suggest another fix: the `experiment`argument should come first in `run_sim.sh`, so that you are always forced to set it and whatever comes next is truly optional. Your string `./run_sim.sh 1 G4p10 Cryostat_K40 100 v3.0.0 head head 0 0 XENONnT` is forcing the `preinit_macro`to be `XENONnT`, which of course makes no sense and brings all the defaults, including the XENON1T detector.